components/sections/migrate/migrated-accounts-rows.tsx

```diff
@@ -50,14 +56,30 @@ const MigratedAccountRows = ({ app, multisigAddresses, destinationAddressesStatu
   const renderStatusIcon = (transaction: Transaction | undefined, hasPendingActions: boolean) => {
     const txStatus = transaction?.status
     const txStatusMessage = transaction?.statusMessage
+    const txDispatchError = transaction?.dispatchError
 
     if (hasPendingActions && !txStatus) {
       return <PendingActionsWarning />
     }
 
     const { statusIcon, statusMessage } = getTransactionStatus(txStatus, txStatusMessage)
 
-    return statusMessage ? <CustomTooltip tooltipBody={statusMessage}>{statusIcon}</CustomTooltip> : statusIcon
+    return statusMessage ? (
```

Comment:
Is it possible we have statusMessage as null, but txDispatchError not null? In that case, the tooltip won't be displayed